reference/outlook/preview/SeriesTime.md

```diff
@@ -0,0 +1,391 @@
+
+##### Requirements
+
+|Requirement|Value|
+|---|---|
+|[Minimum mailbox requirement set version](../tutorial-api-requirement-sets.md)|Preview|
+|[Minimum permission level](https://docs.microsoft.com/outlook/add-ins/understanding-outlook-add-in-permissions)|ReadWriteItem|
+|Applicable Outlook mode|Compose|
+
+##### Examples
+
+This example sets the end date of a recurrence to November 2, 2017.
+
+```js
+var seriesTimeObject = new seriesTime()
+seriesTimeObject.setEndDate(2017, 10, 2);
+```
+
+##### Errors
+
+|Error code|Description|
+|------------|-------------|
+|`Invalid date format`||
+
+#### setEndDate(String date)
+
+Sets the end date of a recurrence pattern.
+
+##### Parameters:
+
+|Name|Type|Attributes|Description|
+|---|---|---|---|
+|`date`|String||End date of the recurrence represented in [ISO 8601](https://www.iso.org/iso-8601-date-and-time-format.html) date standard, that is, "YYYY-MM-DD".|
+
+##### Requirements
+
+|Requirement|Value|
+|---|---|
+|[Minimum mailbox requirement set version](../tutorial-api-requirement-sets.md)|Preview|
+|[Minimum permission level](https://docs.microsoft.com/outlook/add-ins/understanding-outlook-add-in-permissions)|ReadWriteItem|
+|Applicable Outlook mode|Compose|
+
+##### Examples
+
+This example sets the end date of a recurrence to November 2, 2017.
+
+```js
+var seriesTimeObject = new seriesTime()
+seriesTimeObject.setEndDate("2017-11-02");
+```
+
+##### Errors
+
+|Error code|Description|
+|------------|-------------|
+|`Invalid date format`|The date is is not in an acceptable format.|
+
+#### setStartTime(int hours, int minutes)
+
+Sets the start time of the meeting/appointment of a recurrence pattern in whichever time zone the recurrence pattern is set. Default zone is the item's time zone.
+
+##### Parameters:
+
+|Name|Type|Attributes|Description|
+|---|---|---|---|
+|`hours`|Integer||The `hour` portion of the start time. Valid range: 0-24.|
+|`minutes`|Integer||The `minute` portion of the start time. Valid range: 0-59.|
+
+##### Requirements
+
+|Requirement|Value|
+|---|---|
+|[Minimum mailbox requirement set version](../tutorial-api-requirement-sets.md)|Preview|
+|[Minimum permission level](https://docs.microsoft.com/outlook/add-ins/understanding-outlook-add-in-permissions)|ReadWriteItem|
+|Applicable Outlook mode|Compose|
+
+##### Examples
+
+This example sets the start time of a recurrence to 1:30 pm.
+
+```js
+var seriesTimeObject = new seriesTime()
+seriesTimeObject.setStartTime(13, 30);
+```
+
+This example sets the start time of a recurrence to 11:30 am.
+
+```js
+var seriesTimeObject = new seriesTime()
+seriesTimeObject.setStartTime(11, 30);
+```
+
+##### Errors
+
+|Error code|Description|
+|------------|-------------|
+|`Invalid time format`|The time is is not in an acceptable format.|
+
+#### setStartTime(String time)
+
+Sets the start time of the meeting/appointment of a recurrence pattern in whichever time zone the recurrence pattern is set in. Default value is the item's time zone.
```

Comment:
should be consistent across all setTime apis. "Default value is the item's time zone"